### DISCUSSION TITLE
Object Stacking

### DISCUSSION THREAD

**Discussion by Maelle1978** [OrcaSlicer/OrcaSlicer#4433] [Ideas]
I was wondering whether it was possible to stack objects.
Example : 
I've created manually a 5 colours "hueforge" Freddie Mercury
For this, I've downloaded an image I've converted to 5 colours paths in Illustrator.
I've grouped paths correctly to print each other stacked correctly on my Creality Ender 3 V3 SE and exported each colour path to svg file.
Importing each sag file in Orcaslicer, I can decide which thickness I need for each SVG I import.
I can group all imported svg in 1 single objects but then I have to replace all manually to the center of the plate and move each one by one z axis to stack them correctly.
I've done something similar with a Catwoman I had to had a "support" as the model was only place on her boots.
When we group several objects, it would be nice to have the option to stack them automatically on top of each other, it would save time of centering all of them and stack at the correct Z position...

![Capture d’écran 2024-03-12 à 17 37 38](https://github.com/SoftFever/OrcaSlicer/assets/163167648/7bb5ec62-fb89-4e57-bc21-ae3b253b3c23)

![IMG-20240311-WA0032](https://github.com/SoftFever/OrcaSlicer/assets/163167648/82abdcea-35e4-4ed6-bda1-ba7e1c6d8f1f)

**Comment by otede** [comment]
Stacking Objects in Bambu Slicer P1p/P1s?XC1 **NOW CALLED MERGE** - YouTube — https://www.youtube.com/watch?v=_zEYix1Ydlk
by [JCR Builds ](https://www.youtube.com/@jcrbuilds)
